### ISSUE TITLE
Can you give a simple example?

### ISSUE THREAD

**Issue by OliverAbraham** [nikkilocke/NextcloudApi#13]
Hi Nikki,
I'm trying to get your API working.
Your tests are hard to follow.
Is it possible to get a simple example in one method?
That demonstrates how to connect to a Nextcloud instance and list the filenames in the root folder of a user?
I am struggeling with your code and I'm always getting an Exception saying "The format of value '<null>' is invalid"
Best regards!
Oliver

I have tried this:

try
{
    var settings = new Settings()
    { 
        ServerUri = new Uri("http://www.abraham-beratung.de/nextcloud/remote.php/dav"),
        Username = credentials.UserName, 
        Password = credentials.Password 
    };
    var api = new Api(settings);
    var cloudInfo = await CloudFolder.List(api, "files/IT");
}
catch (Exception ex)
{

}

**Comment by nikkilocke** [comment]
Answering this on my phone, a long way from the source code, but I think
the uri should be the Nextcloud server, without remote.php etc.
And, unless you have a folder called files, with a subfolder called IT,
those shouldn't be there either. I can't remember, but you either need an
empty string to get your personal folder, or maybe your user name.
